scripts/run-app.sh

```diff
@@ -18,7 +18,7 @@ export_secrets_from_dir() {
 
     log "Exporting secret '$filename_without_path' as '$secret_name'"
 
-    local secret=$(<$filename);
+    local secret=$(cat $filename);
```

Comment:
Add surrounding quotes to allow for secrets with spaces in them?

Reply:
I'm fixing all shellcheck violations atm, including that :smile: